service/src/main/java/uk/gov/hmcts/reform/fpl/model/ReturnApplication.java

```diff
@@ -17,4 +21,18 @@
     private String submittedDate;
     private String returnedDate;
     private DocumentReference document;
+
+    @JsonIgnore
+    public String getFormattedReturnReasons() {
+        if (reason != null) {
+            String formattedReasons = reason.stream()
+                .map(ReturnedApplicationReasons::getLabel)
+                .map(String::toLowerCase)
+                .collect(Collectors.joining(", "));
+
+            return capitalize(formattedReasons);
```

Comment:
Minor, but do you think it would be more efficient to call `toLowerCase` once just before the `capitalize`, i.e. `capitalize(formattedReasons.toLowerCase());`